src/Hydrators/Family.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Choredo\Hydrators;
+
+use Assert\Assert;
+use Choredo\Entities;
+use Ramsey\Uuid\Uuid;
+use const Choredo\DAYS_OF_WEEK;
+use const Choredo\SHORT_DATA_FIELD_MAX_SIZE;
+
+class Family
+{
+    public function hydrate(array $data): Entities\Family
+    {
+        $data['id'] = $data['id'] ?? Uuid::uuid4();
+        $this->validateData($data);
+
+        return new Entities\Family(
+            $data['id'],
```

Comment:
Family is expecting an UUIDInferface, no? `$data['id']` would be a string unless it's manually hydrated into a UUID object.

Reply:
:+1: